pom.xml

```diff
@@ -46,6 +46,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Is this JUnit dependency required with the new changes?

Reply:
Yes, it is required

Reply:
I'm not sure it is. I can run the tests locally without this dependency...